picard/util/__init__.py

```diff
@@ -469,7 +469,7 @@ def convert_to_string(obj):
 
 
 def htmlescape(string):
-    return cgi.escape(string)
+    return html.escape(string)
```

Comment:
Note that `cgi.escape()` differs from `html.escape()`, `quotes` defaults to `True` instead of `False`.

See https://docs.python.org/3.5/library/cgi.html#cgi.escape and https://docs.python.org/3.5/library/html.html?highlight=html#html.escape

I don't think it has any negative impact, but did you check actual calls to `htmlescape()` ?